src/pages/FrontendMentor/ElvaYBPractice/styles/SinglePriceGridComponent.styled.ts

```diff
@@ -0,0 +1,136 @@
+import styled from 'styled-components'
+
+export const MainPage = styled.div`
+  min-height: 100vh;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: rgba(225, 217, 221, 0.8);
+`
+
+export const MainContainer = styled.div`
+  width: 70%;
+  border-radius: 15px;
+  overflow: hidden;
+`
+export const BottomContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  @media (max-width: 768px) {
+    flex-direction: column;
+  }
+`
+
+export const MiddleContainer = styled.div`
+  color: hsl(179, 62%, 43%);
+  background-color: #ffffff;
+  width: 100%;
+  height: 35vh;
+  @media (max-width: 768px) {
+    display: flex;
+    flex-direction: column;
+    padding: 10%;
+    height: 60vw;
+  }
+`
+
+export const JoinCommText1 = styled.div`
+  color: hsl(179, 62%, 43%);
+  font-size: 30px;
+  font-weight: bold;
+  padding: 30px 00px 10px 80px;
+  @media (max-width: 768px) {
+    font-size: 5vw;
+    padding: 2vw;
+  }
+`
+export const JoinCommText2 = styled.div`
+  color: hsl(71, 73%, 54%);
+  font-size: 16px;
+  font-weight: bold;
+  padding: 00px 00px 10px 80px;
+  @media (max-width: 768px) {
+    font-size: 3vw;
+    padding: 2vw;
+  }
+`
+export const JoinCommText3 = styled.div`
+  color: hsl(218, 22%, 67%);
+  font-size: 16px;
+  padding: 00px 00px 00px 80px;
+  break-between: always;
+  @media (max-width: 768px) {
```

Comment:
I see that you have applied Media query to all the components. It would be good if u can make the parent container implemented with media query and all the children component will follow. Again, this is optional for now.